app-engine/java/resources/META-INF/app-engine-java.xml

```diff
@@ -48,6 +48,11 @@
         </group>
     </actions>
 
+    <module-components>
+        <component>
+            <implementation-class>com.google.cloud.tools.intellij.appengine.java.facet.standard.AppEngineStandardGradleModuleComponent</implementation-class>
```

Comment:
Could this belong to app-engine-java-gradle instead or it's tied to parent app engine java module?

Reply:
good idea. I think that may work. Let me see.

Reply:
this would actually introduce a circular dependency as the app-engine-java module currently uses this directly